### PR TITLE
comment timeout configuration #10

### PR DESCRIPTION
**Pull request by lukaszbanasiak**
issue #10

### CODE REVIEW

File: docs/settings.txt [hunk 1]
```diff
@@ -24,6 +24,13 @@ this will be rejected. Defaults to 3000.
 
 .. setting:: COMMENTS_APP
 
+COMMENT_TIMEOUT
+------------------
+
+The maximum comment form timeout. Defaults is 2 hours.
```
Comment (claudep) [inline docs/settings.txt]: ...timeout in seconds.

Comment (lukaszbanasiak) [inline docs/settings.txt]: @claudep What do you mean? The equation in code gives 2h timeout.

Comment (claudep) [inline docs/settings.txt]: I think it's important to clarify the unit used by the setting, which is seconds in my opinion.

Comment (lukaszbanasiak) [inline docs/settings.txt]: @claudep you're right, fix in 6010e5f

**Comment by claudep** [comment]
Thanks, could you please rebase your patch and merge the commits?

**Comment by claudep** [comment]
Pushed in de6187c6d996

**Pull request closed without merging**